docs/source/_static/main.css

```diff
@@ -0,0 +1,3 @@
+col {
```

Comment:
Maybe this in case there are other kinds of tables elsewhere:
```suggestion
.longtable col {
```